lighthouse-core/gather/gatherers/fonts.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Gatherer = require('./gatherer');
+const fontFaceDescriptors = [
+  'font-family',
+  'font-style',
+  'font-weight',
+  'font-stretch',
+  'unicode-range',
+  'font-variant',
+  'font-feature-settings',
+];
+
+/* eslint-disable */
+function getAllLoadedFonts() {
+  const getFont = fontFace => {
+    const fontRule = {};
+    fontFaceDescriptors.forEach(descriptor => {
+      const descriptorWithoutFont = descriptor.replace('font-', '');
+      fontRule[descriptorWithoutFont] = fontFace[descriptorWithoutFont];
```

Comment:
looking at https://github.com/GoogleChrome/lighthouse/pull/3831/files/bf0d8e3cc3c9a16b93ff26b59048563193db85be#r159987426 it's `fontFace.unicodeRange` and `fontFace.featureSettings` so i think you need to camelize this string too?

Reply:
actually do we ever use the `long-form` string?  actually looks like we don't. It looks like the `fontFaceDescriptors` can be the camelCase `font-`-less strings instead.

Reply:
@paulirish wouldn't it be more declarative to keep it with the css style and just camelcase it?

Reply:
@wardpeet well, right now we never use the css form of these strings, so it seems weird to const them only to translate them for every use.

and TBH i don't think those strings are more canonical in this case as we're in javascript land here where 'font-weight' doesn't exist but `fontWeight` does.